assets/app/views/attachPersistentVolumeClaim.html

```diff
@@ -0,0 +1,198 @@
+                    </span>
+                  </div>
+                </div>
+
+                <div class="form-group">
+                  <label for="volume-name">Volume name</label>
+                  <input
+                      id="volume-path"
+                      class="form-control"
+                      type="text"
+                      name="volumeName"
+                      ng-model="attach.volumeName"
+                      placeholder="(generated if empty)"
+                      autocorrect="off"
+                      autocapitalize="off"
+                      spellcheck="false"
+                      aria-describedby="volume-name-help">
+                  <div>
+                    <span id="volume-name-help" class="help-block">Unique name used to identify this volume. If not specified, a volume name is generated.</span>
+                  </div>
+                </div>
+                <div class="has-error" ng-show="volumeNameAlreadyUsed">
+                  <span class="help-block">
+                    Volume name already exists. Please choose another name.
+                  </span>
+                </div>
+
+                <div ng-show="attach.containers.all">
+                  The volume will be mounted into all containers. You can <a href=""
+                    ng-click="attach.containers.all = false">select specific containers</a> instead.
+                </div>
+
+                <div ng-show="!attach.containers.all" class="form-group">
+                  <h3>Containers</h3>
+                  <div>
+                    <span class="help-block">
+                      Attach the volume to the following containers:
+                    </span>
+                  </div>
+                  <div class="checkbox" ng-repeat="container in (attach.deployment.spec.template.spec.containers || attach.deploymentConfig.spec.template.spec.containers) track by (container | uid)">
+                    <label>
+                      <input type="checkbox" ng-model="attach.containers.individual[container.name]">
+                      <b>{{container.name}}</b> from image <i>{{container.image}}</i>
+                    </label>
+                  </div>
+                  <div class="has-error" ng-show="!containerToAttachProvided()">
+                    <span class="help-block">
+                      You must select at least one container.
+                    </span>
+                  </div>
+                </div>
+
+                <div class="button-group gutter-top gutter-bottom">
+                  <button type="submit"
+                      class="btn btn-primary btn-lg"
+                      ng-click="attachPersistentVolumeClaim()"
+                      ng-disabled="attachPersistentVolumeClaimForm.$invalid || disableInputs || !attachPersistentVolumeClaim || !containerToAttachProvided()"
+                      value="">Attach</button>
+                  <a class="btn btn-default btn-lg" href="#" back>Cancel</a>
```

Comment:
I was thinking about someone clicking Submit then Cancel while we're still processing the form, but maybe not a problem in practice.